### DISCUSSION TITLE
Nub question: Why I can't find the doc of "vn.envUint" when I google it?

### DISCUSSION THREAD

**Discussion by CH-coding-fire** [Cyfrin/foundry-full-course-cu#1879] [General]
As i asked, why?
![image](https://github.com/Cyfrin/foundry-full-course-cu/assets/67637342/627b5963-2ed9-4e0f-840f-32995d667999)
So I need to search official doc myself: https://book.getfoundry.sh/cheatcodes/env-uint

I find this very inefficient, I wonder is there some better way or the configuration of my Google has something wrong?

**Comment by EngrPips** [comment]
`vm.envUnit` is a cheat code used in `Foundry` to read an environment variable. So, if you want to read anything about It, the `Foundry` docs is the right place to look. In summary, the cheat code and other `vm` cheat codes you use while developing smart contracts with `Foundry` are peculiar to `Foundry.`
